lib/recaptcha/client_helper.rb

```diff
@@ -23,7 +26,10 @@ def recaptcha_tags(options = {})
       data_attributes[:sitekey] = public_key
       data_attributes = data_attributes.map { |k, v| %(data-#{k.to_s.tr('_', '-')}="#{v}") }.join(" ")
 
-      html = %(<script src="#{script_url}" async defer></script>\n)
+      data_attributes << %( id="#{options[:id]}") if options[:id]
```

Comment:
it's not a data_attribute, so this looks a bit strange ... add it straight into the html below ?